doc/sphinx-guides/source/api/client-libraries.rst

```diff
@@ -50,6 +50,8 @@ Python
 
 There are multiple Python modules for interacting with Dataverse APIs.
 
+`UBC's Dataverse Utilities <https://ubc-library-rc.github.io/dataverse_utils/>`_ are a set of Python console utilities which allow one to upload datasets from a tab-separated-value spreadsheet, bulk release multiple datasets, bulk delete unpublished datasets, quickly duplicate records. replace licenses, and more. For additional information see their `PyPi page <https://pypi.org/project/dataverse-utils/>`_.
```

Comment:
https://dataverse-guide--9929.org.readthedocs.build/en/9929/api/client-libraries.html looks good but I'm not so sure these utilities should be at the top. How about between pyDataverse and dataverse-client-python?

Reply:
Fixed in a1a1233. Thanks, @donsizemore!